sky/resources.py

```diff
@@ -1,5 +1,6 @@
 """Resources: compute requirements of Tasks."""
```

Comment:
We should also fix the `less_demanding_than` function, so that a job requires `high` disk type will not be able to run on the cluster with `low` disk type.